### ISSUE TITLE
Reproducing paper results

### ISSUE THREAD

**Issue by Radu1999** [GuyTevet/MotionCLIP#32]
Was clip finetuned in training the checkpoint? I am unable to reproduce results with clip frozen (didnt try with it unfrozen yet).

**Comment by GuyTevet** [comment]
The CLIP model was frozen indeed. I can think of two possible issues, (1) MotionCLIP is seed sensitive - try to train it with a few seeds, and (2) the text labels (class names) are not aligned with the ones we used. Anyway, I recommend using the pre-trained model.
